draft-ietf-quic-recovery.md

```diff
@@ -228,6 +228,25 @@ QUIC endpoints measure the delay incurred between when a packet is received and
 when the corresponding acknowledgment is sent, allowing a peer to maintain a
 more accurate round-trip time estimate (see Section 13.2 of {{QUIC-TRANSPORT}}).
 
+### Probe Timeout Replaces RTO and TLP
+
+QUIC uses a probe timeout (see {{pto}}), with a timer based on TCP's RTO
+computation.  QUIC's PTO includes the peer's maximum expected acknowledgement
+delay instead of using a fixed minimum timeout. Unlike TCP, which collapses
+the congestion window upon expiry of an RTO, QUIC does not collapse the
+congestion window until persistent congestion ({{persistent-congestion}}) is
+declared and instead allows probe packets to temporarily exceed the congestion
+window whenever the timer expires.  In practice, this is similar to TCP with
+F-RTO, but it does allow more packets to be sent when the congestion window was
+not fully utilized prior to the probe timeout expiring. Though this is slightly
```

Comment:
```suggestion
window whenever the timer expires.  In comparison to TCP with F-RTO
{{!RFC5682}}, which corrects an unnecessary collapse of the congestion window,
this design aims to prevent the window from collapsing. Though this is slightly
```

Perhaps?